docs/getting-started/install/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Install Shopify CLI
+section: getting-started
+toc: false
+redirect_from: "/install/"
+---
+
+Shopify CLI can be installed using a variety of package managers.
+
+### Ruby gem
+
+```console
+$ gem install shopify-cli
+```
+
+### apt (Debian, Ubuntu)
+
+Currently, you’ll need to install a downloaded .deb file with an explicit version number. Make sure you check the [release page](#) to make sure you install the latest package.
```

Comment:
TODO: need the URL where users can find the .deb releases

Reply:
On github that URL would be https://github.com/Shopify/shopify-app-cli/releases